src/main/resources/vue/components/_charts.vue

```diff
@@ -95,7 +95,7 @@
         });
 
         function createColorArray(length) {
-            const colors = ["#54ca76", "#f5c452", "#f2637f", "#9261f3", "#31a4e6", "#55cbcb"];
+            const colors = ["#1e81b0", "##eeeee4", "##e28743", "##eab676", "##76b5c5", "##21130d", "#873e23", "#d5817c"];
```

Comment:
A few color strings here are invalid due to the double hash prefix: `##`

Also please include a screenshot (see: https://github.com/tipsy/profile-summary-for-github/blob/master/Documentation/Building.md)